### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#108507] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I apologize for the inconvenience caused by the account suspension. I sincerely apologize to everyone for posting content related to Copilot that I now realize may have violated community guidelines. I deeply regret my actions and have been repenting for a long time.

Now my problem is, if the account has been blocked, then it is impossible to log in to the GitHub Support platform, so there is no way to appeal through this account, but the platform requires me to use the banned account to submit a work order：

The account associated with the email address you've contacted us from is not blocked.

If you are inquiring about another account that you own, could you please write in from the verified email address associated with it? You can forward this message to that email address and reply from there.

Once we're in contact via a verified email address, we'll try to assist further.
Regards,
Dash
GitHub Support

This has become some kind of paradox. What should I do? My banned account is github.com/zcxey2911  zcxey2911@outlook.com

the ticket is #2588806

<img width="837" alt="PixPin_2024-02-16_22-14-07" src="https://github.com/community/community/assets/159596865/17c03d71-c4e8-4956-acd7-f1f7d938d42a">

**Comment by carlosmintfan** [comment, marked as answer]
Hi @lqxcip2911, I'm not a GitHub support member but I think you should open your email inbox of the email of the GitHub account you wrote to them (possibly your @lqxcip2911 account), search for the email with the same content as the screenshot you showed, forward it to your zcxey2911 email and reply from there as an email, not from the GitHub support platform.

**Reply by ghost** [reply]
thanks a lot
